alert.go

```diff
@@ -442,7 +476,11 @@ func (a *AlertNode) runAlert([]byte) error {
 			var highestPoint *models.BatchPoint
 
 			for i, p := range b.Points {
-				l := a.determineLevel(p.Time, p.Fields, p.Tags)
+				var currentLevel AlertLevel
+				if _, ok := a.states[b.Group]; ok {
+					currentLevel = a.states[b.Group].currentLevel()
```

Comment:
Here and above instead of assigning the result to `_` and then doing another map lookup simply keep the state var around.

```go
if state, ok := a.states[b.Group]; ok {
	currentLevel = state.currentLevel()
}
```